model_one/cli.py

```diff
@@ -1,198 +1,274 @@
-from array import array
+import os
 import json
-from re import M
-from time import time
 import requests
-import os.path
-import urllib.parse
 
-import os
+from enum import Enum
+from functools import wraps
+from typing import Optional, List, Union
+
+from pandas import Series
+
 
 API_KEY = os.environ.get("BEYONDML_API_KEY")
-api_url = 'https://api.beyond.ml'
+MINIMUM_ENTRIES = 32
```

Comment:
That's the former magic constant
It was not that magic at all, i guess it ok to have a minimal number of entries in an uploaded dataset